src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,40 @@
+.clock {
+  position: relative;
+
+  width: $clock-size;
+  height: $clock-size;
+  border: 1px solid #000;
+
+  background: url(../../src/images/1602313045_1.jpg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+
+  &__arrow {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateY(-50%) rotate(-90deg);
+    transform-origin: center left ;
+
+    width: $clock-arrow-size;
+    height: 5px;
+
+    background-color: #000;
+    border-radius: 5px;
+
+    animation-name: move;
+    animation-duration: $duration-of-arrorw;
+    animation-timing-function: linear;
+  }
+}
+
+@keyframes move {
+  from {
+    transform: rotate(-90deg);
+  }
+
+  to {
+    transform: rotate(270deg);
+  }
+}
```

Comment:
You can move all keyframes to separate file

Reply:
I'd like to leave the animation here cause it helps me to understand with which element I use the animation